backend/domain/Enum/Role.js

```diff
@@ -1,6 +1,12 @@
 // Enums for Roles.
-// Can be developed into classes if functionality is required.  
-const RoleEnum = { "CIVILIAN": 1, "MAFIA": 2, "MEDIC": 3, "DETECTIVE": 4 }
-Object.freeze(RoleEnum)
+// Can be developed into classes if functionality is required.
+const RoleEnum = {
+  CIVILIAN: "civilian",
+  MAFIA: "mafia",
+  MEDIC: "medic",
+  DETECTIVE: "detective",
+  TANNER: "tanner",
```

Comment:
I thought we decided on Jester instead of Tanner?

Reply:
Yup, I've changed it now